docs/src/snap/explanation/certificates.md

```diff
@@ -0,0 +1,59 @@
+# Certificates
+
+Certificates are a crucial part of Kubernetes' security infrastructure, serving
+to authenticate and secure communication within the cluster. They play a key
+role in ensuring that communication between various components (such as the
+API server, kubelets, and the datastore) is both encrypted and restricted to
+authorised components only.
+
+In Kubernetes, [X.509][] certificates are primarily used for
+[Transport Layer Security][] (TLS), securing the communication between the
+cluster's components.
+
+## What Is a Certificate Refresh?
+
+A certificate refresh in {{product}} refers to the process of renewing or
```

Comment:
Is there a difference between renewing and rotating certificates?

Reply:
We call it refresh (e.g., `refresh-certs`), but the term is interchangeable with renew or rotate.